daemon/src/maker_inc_connections.rs

```diff
@@ -186,6 +192,15 @@ impl Actor {
             }
         }
     }
+
+    async fn handle_read_fail(&mut self, msg: ReadFail, _: &mut Context<Self>) {
```

Comment:
The macro can handle fn definitions without `Context`.

```suggestion
    async fn handle_read_fail(&mut self, msg: ReadFail) {
```

Reply:
Nice!